docs/developer-setup/README.md

```diff
@@ -31,9 +31,6 @@
 
 ## Prerequisites
 
-- Install Nvidia Omniverse: https://www.nvidia.com/en-us/omniverse/download/
-- Install Omniverse Create 2022.3.3 (or later)
```

Comment:
I removed these steps since they aren't actually required for building and running Cesium for Omniverse.

I think there should be a separate section that goes over how to load the extension in Code or USD Composer but it could be a separate PR.